### PR TITLE
Integrate with mixer client.

### PR DESCRIPTION
**Pull request by qiwzhang**

### CODE REVIEW

File: src/envoy/prototype/server_config.pb.txt [hunk 1]
```diff
@@ -2,5 +2,5 @@ cloud_tracing_config {
   force_disable: true
 }
 mixer_options {
-  mixer_server: "mixer_server"
-}
\ No newline at end of file
+  mixer_server: "localhost:9091"
```
Comment (kyessenov) [inline src/envoy/prototype/server_config.pb.txt]: So is cluster definition not needed? What about DNS?

File: WORKSPACE [hunk 2]
```diff
@@ -16,29 +16,42 @@
...
+    name = "protobuf_bzl",
+    # Injecting an empty BUILD file to prevent using any build target
+    build_file_content = "",
+    commit = "05090726144b6e632c50f47720ff51049bfcbef6",
+    remote = "https://github.com/google/protobuf.git",
+)
 
 load(
-    "//contrib/endpoints:repositories.bzl",
+    "@mixerclient_git//:repositories.bzl",
```
Comment (lizan) [inline WORKSPACE]: load as little as possible from `@mixerclient_git//:repositories.bzl`

BoringSSL has to be compatible with the version required from envoy, mixerclient has little thing to do with version compatibilities. same for grpc/protobuf/googletest/googleapis.

File: contrib/endpoints/repositories.bzl [hunk 3]
```diff
@@ -14,314 +14,8 @@
 #
 ################################################################################
 #
-def zlib_repositories(bind=True):
```
Comment (lizan) [inline contrib/endpoints/repositories.bzl]: don't remove those, per reason stated above.

File: src/envoy/prototype/server_config.pb.txt (hunk 1 above)
Comment (qiwzhang) [inline src/envoy/prototype/server_config.pb.txt]: Yes, we don't need cluster for Mixer server.  Proxy is using grpc Channel, could  not use Envoy cluster manager. 

I think we can specify DNS to grpc channel. it should do DNS lookup.

Comment (lizan) [inline src/envoy/prototype/server_config.pb.txt]: the approach to use grpc lib doesn't require cluster definitions, full stack are handled in grpc lib.

File: WORKSPACE (hunk 2 above)
Comment (qiwzhang) [inline WORKSPACE]: Done

File: contrib/endpoints/repositories.bzl (hunk 3 above)
Comment (qiwzhang) [inline contrib/endpoints/repositories.bzl]: Done

File: src/envoy/prototype/server_config.pb.txt (hunk 1 above)
Comment (kyessenov) [inline src/envoy/prototype/server_config.pb.txt]: Ok, might want to explain whether GRPC does DNS resolution on every request or if it is blocking.

**Pull request merged**